package.json

```diff
@@ -50,7 +50,7 @@
 		"remark-cli": "^7.0.1",
 		"remark-preset-lint-consistent": "^2.0.3",
 		"remark-preset-lint-recommended": "^3.0.3",
-		"stylelint": "^12.0.0",
+		"stylelint": "^13.1.0",
```

Comment:
Should be added to `peerDependencies` as well: `"^10.1.0 || ^11.0.0 || ^12.0.0 ||  ^13.0.0"`

Reply:
Thanks, fixed in https://github.com/WordPress-Coding-Standards/stylelint-config-wordpress/commit/c2c1112b9631c10d00d61d87a8beba8a715ea167